assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,146 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
```

Comment:
I'm pretty sure there is no need for `role="navigation"` on a nav element anymore https://www.w3.org/TR/html5/dom.html#sec-strong-native-semantics